packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -216,85 +217,72 @@ export class SelectFileName
   }
 }
 
-export abstract class SelectDirPath
+export class SelectOutputDir
   implements ParametersGatherer<{ outputdir: string }> {
-  private explorerDir: string | undefined;
-  private globKeyWord: string | undefined;
-
-  public constructor(explorerDir?: vscode.Uri, globKeyWord?: string) {
-    this.explorerDir = explorerDir ? explorerDir.fsPath : explorerDir;
-    this.globKeyWord = globKeyWord;
+  private typeDir: string;
+  private typeDirRequired: boolean | undefined;
+  public readonly defaultOutput = '/main/default';
```

Comment:
I think this is going to break on Windows.

Reply:
Yikes good catch